test/fixtures/logbucket-folder/main.tf

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "log_export" {
+  source             = "../../../examples/logbucket/folder"
+  parent_resource_id = var.parent_resource_folder
+  project_id         = var.project_id
+}
```

Comment:
Our new approach is to not have fixtures if possible for new tests. In this case, fixture is a thin wrapper for example and can be removed. The test framework can directly actuate the example.

Reply:
The issue was that the go test looks for a fixture and/or example only one level down from `fixtures/` or `examples/`. Since our examples are two levels down, the framework wasn't picking it up. Is there a way to  specify where the config is for the test to call?

Reply:
Yeah you can use the withTFDir option
https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft#WithTFDir

Reply:
@g-awmalik lets switch to using this as we shouldn't need a fixture.

Reply:
ack